.github/workflows/build.yml

```diff
@@ -67,7 +67,8 @@ jobs:
           path: build/reports/tests/test/
 
   mac-image:
-    name: Tower CLI build Mac native image
+    name: MAC
```

Comment:
Does GH already have support for M1?

Reply:
No yet. They're planning a preview 2022: https://github.com/actions/virtual-environments/issues/2187